### PR TITLE
feat(step): add table output

### PR DESCRIPTION
**Pull request by jbrockopp**
* Added a `table()` function to `go-vela/cli/action/step`
* Added tests for the `table()` function
* Added a `wideTable()` function to `go-vela/cli/action/step`
* Added tests for the `wideTable()` function
* Added `go-vela/cli/internal/output.Dump` as an output option
* Added `go-vela/cli/internal/output.Spew` as an output option

### CODE REVIEW

File: action/step/get.go [hunk 1]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
 
 	// handle the output based off the provided configuration
 	switch c.Output {
-	case "json":
-		// output the step in JSON format
+	case output.DriverDump:
```
Comment (jbrockopp) [inline action/step/get.go:29]: Each function now supports the `dump` driver from `go-vela/cli/internal/output`

File: action/step/get.go [hunk 2]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
-		// 	return err
-		// }
-	case "yaml":
-		// output the step in YAML format
+		// output the steps in wide table format
+		err := wideTable(steps)
+		if err != nil {
+			return err
+		}
+	case output.DriverSpew:
```
Comment (jbrockopp) [inline action/step/get.go:47]: Each function now supports the `spew` driver from `go-vela/cli/internal/output`

File: action/step/get.go [hunk 3]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
-		// TODO: create output.Wide function
-		//
-		// err := output.Wide(steps)
-		// if err != nil {
-		// 	return err
-		// }
-	case "yaml":
-		// output the step in YAML format
+		// output the steps in wide table format
+		err := wideTable(steps)
```
Comment (jbrockopp) [inline action/step/get.go:43]: Adding support for `wide` output option with a new `wideTable()` function

File: action/step/get.go [hunk 4]
```diff
@@ -26,28 +26,39 @@ func (c *Config) Get(client *vela.Client) error {
...
+		// output the steps in YAML format
 		err := output.YAML(steps)
 		if err != nil {
 			return err
 		}
 	default:
-		// output the step in default format
-		err := output.Default(steps)
+		// output the steps in table format
+		err := table(steps)
```
Comment (jbrockopp) [inline action/step/get.go:61]: Updated the default output option with a new `table()` function

File: action/step/table.go [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+	"github.com/go-vela/types/library"
+
+	"github.com/dustin/go-humanize"
+	"github.com/gosuri/uitable"
+)
+
+// table is a helper function to output the
+// provided steps in a table format with
+// a specific set of fields displayed.
+func table(steps *[]library.Step) error {
```
Comment (jbrockopp) [inline action/step/table.go:24]: This is where we add the new `table()` function

File: action/step/table.go [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+		return err
+	}
+
+	return nil
+}
+
+// wideTable is a helper function to output the
+// provided steps in a wide table format with
+// a specific set of fields displayed.
+func wideTable(steps *[]library.Step) error {
```
Comment (jbrockopp) [inline action/step/table.go:58]: This is where we add the new `wideTable()` function

File: action/step/table_test.go [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+	s.SetRepoID(1)
+	s.SetNumber(1)
+	s.SetName("clone")
+	s.SetImage("target/vela-git:v0.3.0")
+	s.SetStatus("running")
+	s.SetExitCode(0)
+	s.SetCreated(1563474076)
+	s.SetStarted(1563474078)
+	s.SetFinished(1563474079)
+	s.SetHost("example.company.com")
```
Comment (wass3r) [inline action/step/table_test.go:115]:
```suggestion
	s.SetHost("host.example.com")
```

**Pull request merged**